src/views/partials/chat.hbs

```diff
@@ -0,0 +1,34 @@
+<div class="chat">
+  <div class="messages__container">
+    <ul class="messages">
+      <li> {{username}}: {{message}} </li>
```

Comment:
Also, no need for this `<li> {{username}}: {{message}} </li>` here, as messages will get dynamically injected